src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+        super(ColorSupplier.getRandomColor());
+        radius = FigureSupplier.getRandomLength();
```

Comment:
It's better to pass values from outside rather than creating a shape with random values in the shape itself. Redo it in all shapes.